common/controllers/AuthController.ts

```diff
@@ -283,10 +348,31 @@ export default abstract class AuthControllerBase implements IAuthController {
         }
     }
 
-    async updatePassword(password: string, oldPassword?: string): Promise<AuthResult> {
```

Comment:
Is it possible to create a new function rather than updating that?

Reply:
Done.